server/belga/io/feed_parsers/belga_newsml_1_2.py

```diff
@@ -169,6 +197,10 @@ def parser_newsitem(self, newsitem_el):
                 # NewsComponent
                 try:
                     self.parser_newscomponent(item, news_component_2)
+                    # SDBELGA-328
+                    if item.get('abstract'):
+                        abstract = '<p>' + item['abstract'] + '</p>'
+                        item['body_html'] = abstract + item['body_html']
```

Comment:
From the ticket, this is Belga Remote specific logic, right?

Reply:
I've clarified it with Stefaan and it should be applied to all belga news ml inputs.